src/opentimelineview/track_widgets.py

```diff
@@ -230,6 +233,39 @@ def __init__(self, *args, **kwargs):
         self.source_name_label.setText('GAP')
 
 
+class TrackNameItem(BaseItem):
+
+    def __init__(self, track, rect, *args, **kwargs):
+        super(TrackNameItem, self).__init__(None, None, rect,
+                                            *args, **kwargs)
+        track_name = 'Track' if track.name == '' else track.name
+        if len(track_name) > 7:
+            track_name = track_name[:7] + '...'
```

Comment:
I'll make the changes suggested above as they aren't major changes. Can this come in another PR. I have another PR on top of this. Although it's unrelated, I just don't want to break anything, just in case.

Reply:
Sure, makes sense.  Thanks!

Reply:
Actually I haven't opened that PR yet.
I've written about it in #280. Could you please take a look?